phys/module_mp_p3.F

```diff
@@ -3436,10 +4087,10 @@ SUBROUTINE p3_main(qc,nc,qr,nr,th_old,th,qv_old,qv,dt,qitot,qirim,nitot,birim,ss
 !  note: This is not necessary for GEM, which already has these values available
 !        from the beginning of the model time step (TT_moins and HU_moins) when
 !        s/r 'p3_wrapper_gem' is called (from s/r 'condensation').
-  if (trim(model) == 'WRF') then
+!!  if (trim(model) == 'WRF') then
```

Comment:
If these are note required by GEM, then shouldn't there be an IF test? If no IF test, is the comment wrong?

Reply:
*not*